landoapi/phabricator_client.py

```diff
@@ -0,0 +1,109 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import requests
+
+
+class PhabricatorClient:
+    """ A class to interface with Phabricator's Conduit API. 
+    
+    All request methods in this class will throw a PhabricatorAPIException if 
+    Phabricator returns an error response. If there is an actual problem with
+    the request to the server or decoding the JSON response, this class will
+    bubble up the exception. These exceptions can be one of the request library
+    exceptions or a JSONDecodeError.
+    """
+
+    def __init__(self, api_key):
+        self.api_url = os.getenv('PHABRICATOR_URL') + '/api'
+        self.api_key = api_key
+
+    def get_revision(self, id=None, phid=None):
+        """ Gets a revision as defined by the Phabricator API.
+        
+        Args:
+            id: The id of the revision if known. This can be in the form of 
+                an integer or an integer prefixed with 'D', e.g. 'D12345'.
+            phid: The phid of the revision to be used if the id isn't provided. 
+        
+        Returns: 
+            A hash of the revision data just as it is returned by Phabricator.
+            Returns None, if the revision doesn't exist, or if the api key that 
+            was used to create the PhabricatorClient doesn't have permission to 
+            view the revision. 
+        """
+        if id:
+            id_num = str(id).strip().replace('D', '')
+            result = self._GET('/differential.query', {'ids[]': [id_num]})
+        elif phid:
+            result = self._GET('/differential.query', {'phids[]': [phid]})
+        else:
```

Comment:
Instead of this `else`, can you just initialize `result` to `None`?  Wont the return ternary take care of the return value?